package/MDAnalysis/analysis/encore/dimensionality_reduction/__init__.py

```diff
@@ -0,0 +1 @@
+from .stochasticproxembed import StochasticProximityEmbedding
```

Comment:
is this the only line in the whole folder? If can move it up to the actual files where you use this.